src/confluent_kafka/src/Consumer.c

```diff
@@ -1039,13 +1119,20 @@ static PyMethodDef Consumer_methods[] = {
 	  "  :param callable on_revoke: callback to provide handling of "
 	  "offset commits to a customized store on the start of a "
 	  "rebalance operation.\n"
+	  "  :param callable on_lost: callback to provide handling in "
+	  "the case a partition assignment has been lost. If not "
+	  "specified, lost partition events will be delivered to "
+	  "on_revoke, if specified. Partitions that have been lost may "
+	  "already be owned by other members in the group and therefore "
+	  "commiting offsets, for example, may fail.\n"
```

Comment:
committing.

And should the recommendation be not to commit offsets perhaps?

Reply:
i think that is implicit. btw: i copy/pasted this sentence from the go impl. that you did.